docs/products/clickhouse/concepts/data-integration-overview.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Aiven for ClickHouse® data service integration
+sidebar_label: Data integration
+keywords: [data service integration, data source integration, managed credentials integration, managed databases integration, named collections]
+---
+
+Aiven for ClickHouse® supports different types of integration allowing you to efficiently connect with other services or data sources and access the data to be processed.
+
+There are a few ways of classifying integration types supported in Aiven for ClickHouse:
+
+- [By purpose](/docs/products/clickhouse/concepts/data-integration-overview#observability-integrations-vs-data-source-integrations):
+  observability integration vs data source integration
+- [By location](/docs/products/clickhouse/concepts/data-integration-overview#in-aiven-integrations-vs-external-integrations):
+  in-aiven integration vs external integration
+- By scope: [managed databases integration](/docs/products/clickhouse/concepts/data-integration-overview#managed-databases-integration) and
+  [managed credentials integration](/docs/products/clickhouse/concepts/data-integration-overview#managed-credentials-integration)
+
+## Observability integrations vs data source integrations
+
+Aiven for ClickHouse supports observability integrations and data source integrations,
+which have different purposes:
+
+- [Observability integration](/docs/products/clickhouse/howto/list-integrations) is
+  connecting to other services (either in-Aiven or external) to expose and process logs
+  and metrics.
+- Data service integration is connecting to other services (either in-Aiven or external) to
+use them as data sources.
+
+## In-Aiven integrations vs external integrations
+
+By enabling data service integrations, you create streaming data pipelines across
+services. Depending on where the services are located, you can have either in-Aiven
+integrations (between Aiven services) or external integrations (between an Aiven service
+and an external data source or application).
+
+For integrating with external data sources, Aiven for ClickHouse provides two types of
+data service integrations:
+
+- [Managed databases](/docs/products/clickhouse/concepts/data-integration-overview#managed-databases-integration)
```

Comment:
The term `Managed databases` came up again in our daily. Its sounds a bit generic, wdyt?

cc: @aris-aiven @kmichel-aiven

Reply:
Do we have any better ideas? auto-generated databases? auto-created databases?

Reply:
Lets wait for an additional [response](https://aiven-io.slack.com/archives/C02H4MQF4VC/p1721642422723329?thread_ts=1721216275.612719&cid=C02H4MQF4VC), otherwise the name should be fine.